src/App.jsx

```diff
@@ -1,24 +1,61 @@
-import {useEffect, useState} from 'react'
+import {useEffect, useRef, useState} from 'react'
 import { SignClient } from '@walletconnect/sign-client';
 import { Web3Modal } from '@web3modal/standalone';
 import './App.css'
 
 
 // Your dapp's Project ID from https://cloud.walletconnect.com/
-const WC_PROJECT_ID = '9e1152b9dc0318eea105dc31238fbc00';
+const WC_PROJECT_ID = 'cc3b2ecffec2ba45dfa80295a62b3f5a';
```

Comment:
@tvd12 any reason to change this? I think the original one is correct. it's ID of this dapp, not the wallet's id. right?

Reply:
Ok, I will revert it